### PR TITLE
Enable Bulgarian i18n

### PR DESCRIPTION
**Pull request by fintara**
#1442 
Some translations were also fixed/improved in particular ordinal numbers and parsing.
Couldn't make it parse day periods (morning, evening, etc).

### CODE REVIEW

File: src/locale/bg/_lib/localize/index.js [hunk 1]
```diff
@@ -1,54 +1,130 @@
...
+    noon: 'на обяд',
+    morning: 'сутринта',
+    afternoon: 'следобед',
+    evening: 'вечерта',
+    night: 'през нощта'
+  }
 }
 
-function ordinalNumber (dirtyNumber) {
+function ordinalNumber(dirtyNumber, dirtyOptions) {
```
Comment (tunnckoCore) [inline src/locale/bg/_lib/localize/index.js]: I'm feeling that definitely something is wrong here. Can check more deeply locally, later tonight.

**Comment by tunnckoCore** [comment]
Actually, it basically is the same as `mk` - derived from bulgarian, mixed with serbian feelings and sounds, because the Yugoslavian times.

So

```
function ordinalNumber (dirtyNumber) {
  var number = Number(dirtyNumber)

  var rem100 = number % 100
  if (rem100 > 20 || rem100 < 10) {
    switch (rem100 % 10) {
      case 1:
        return number + '-ви'
      case 2:
        return number + '-ри'
      case 7:
      case 8:
        return number + '-ми'
    }
  }
  return number + '-ти'
}
```

**Comment by kossnocorp** [comment]
@tunnckoCore thank you for your input!

File: src/locale/bg/_lib/localize/index.js (hunk 1 above)
Comment (fintara) [reply src/locale/bg/_lib/localize/index.js]: @tunnckoCore is it the 7 & 8 that looked strange to you, or the different `ending`s?

Comment (tunnckoCore) [reply src/locale/bg/_lib/localize/index.js]: Should be as it was. I posted a comment with the function as I think it should be.

Comment (fintara) [reply src/locale/bg/_lib/localize/index.js]: Well, in Bulgarian nouns can be masculine, feminine or neuter. Saying "2002-ри година" is incorrect, since година (year) is feminine, hence it should be "2002-ра година", same with minutes and seconds. 
The previous function did not take 7 & 8 into account, and instead returned suffix `-и` which turns out is grammatically correct but debatable (most people, including official labels use -ти/-та everywhere). 
The function in your comment does not take into account noun's gender.

So if there has to be a change, it would be replacing -ти/-та with -и/-а at the end.

Comment (tunnckoCore) [reply src/locale/bg/_lib/localize/index.js]: Yea. But it depends.

**Comment by tunnckoCore** [comment]
check https://github.com/moment/moment/blob/develop/locale/bg.js#L62

**Comment by fintara** [comment]
It turns out it's not such an easy matter. According to https://www.balgarskiezik.eu/2-2017/VMurdarov_EZ-KULTURA.pdf my latest commit is correct, minus 100 and 1000s cases.
Just to be clear, are we to ignore noun genders? Russian locale, for example, does not.

File: src/locale/bg/snapshot.md [hunk 2]
```diff
@@ -23,29 +23,29 @@
...
 |                                 | LLLL         | 2019-02-11T12:13:14.015Z | февруари                                   | 2019-02-01T00:00:00.000Z |
 |                                 |              | 2019-07-10T12:13:14.015Z | юли                                        | 2019-07-01T00:00:00.000Z |
 |                                 | LLLLL        | 2019-02-11T12:13:14.015Z | февруари                                   | 2019-02-01T00:00:00.000Z |
 |                                 |              | 2019-07-10T12:13:14.015Z | юли                                        | 2019-07-01T00:00:00.000Z |
 | Local week of year              | wo           | 2019-01-01T12:13:14.015Z | 1-ви                                       | 2018-12-31T00:00:00.000Z |
-|                                 |              | 2019-12-01T12:13:14.015Z | 48-ми                                      | 2019-11-25T00:00:00.000Z |
+|                                 |              | 2019-12-01T12:13:14.015Z | 48-и                                       | 2019-11-25T00:00:00.000Z |
```
Comment (tunnckoCore) [inline src/locale/bg/snapshot.md]: Nope.

### week of year

`1-ва (седмица)` - `1st`
`2-ра (седмица)` - `2nd`
`3-тa (седмица)` - `3rd`
`4-тa (седмица)` - `4th`
`5-тa (седмица)` - `5th`
`7-ма (седмица)` - `7th`
`8-ма (седмица)` - `8th`
`9-та (седмица)` - `9th`
`10-та (седмица)`
`11-та (седмица)`
`12-та (седмица)`
`13-та (седмица)`
`21-ва (седмица)` - `21th`
`22-ра (седмица)` - `22nd`
`23-та (седмица)` - `23rd`

### minutes / seconds

If you want to say

- "1 minute and 1 second": _1 минута и 1 секун**да**_
- "1 minute and 2/3/7/8/10/11/17/23 seconds": _1 минута и 2/3/7/8/10/11/17/23 секун**ди**_
- "2 minutes and 1 second": _2 мину**та** и 1 секун**да**_
- "2 minutes and 2/13/27 seconds": _2 мину**ти** и 2/13/27 секун**ди**_


### day of month / year

`1-ви ден` / `1-ви Март`
`2-ри ден`
`3-ти ден` / `1-ти Март
`7-ми ден`
`8-ми ден`
`9-ти ден`
`11-ти ден`

`20-ти ден`
`21-ви ден`
`22-ри ден` / `22-ри Септември`
`23-ти ден`
`27-ми ден`
`28-ми ден`
`29-ти ден`

`30-ти ден`
`31-ви ден` / `31-ви Август` (
`32-ри ден`
`33-ти ден`
`37-ми ден`
`38-ми ден`
`39-ти ден`


...

Basically with the `t` (т) and `m` (м)


`150-ти ден`
`151-ви ден`
`152-ри ден`
`153-ти ден`
`157-ми ден`
`158-ми ден`
`159-ти ден`

**Comment by tunnckoCore** [comment]
Mostly. 

Some examples in the commit comment.

**Comment by fintara** [comment]
With the latest commit:

0-ев ден
0-ва седмица
1-ви, 2-ри, 3-ти, 7-ми месец
1-ва, 2-ра, 3-та, 7-ма минута
100-тен ден, 100-тна секунда
123-ти ден, 123-та секунда
1000-ен ден, 1000-на седмица
1234-ти ден, 1234-та седмица

I think this is most accurate and realistic.

**Comment by tunnckoCore** [comment]
Sure. It's okay to me now too :)
